packages/template-retail-react-app/app/components/_app-config/index.jsx

```diff
@@ -37,18 +43,39 @@ const AppConfig = ({children, locals = {}}) => {
     const [basket, setBasket] = useState(null)
     const [customer, setCustomer] = useState(null)
 
+    const {correlationId} = useCorrelationId()
+    const headers = {
+        'correlation-id': correlationId
+    }
+
+    const commerceApiConfig = locals.appConfig.commerceAPI
+
+    const appOrigin = getAppOrigin()
+
     return (
-        <MultiSiteProvider site={locals.site} locale={locals.locale} buildUrl={locals.buildUrl}>
-            <CommerceAPIProvider value={locals.api}>
-                <CustomerProvider value={{customer, setCustomer}}>
-                    <BasketProvider value={{basket, setBasket}}>
-                        <CustomerProductListsProvider>
-                            <ChakraProvider theme={theme}>{children}</ChakraProvider>
-                        </CustomerProductListsProvider>
-                    </BasketProvider>
-                </CustomerProvider>
-            </CommerceAPIProvider>
-        </MultiSiteProvider>
+        <CommerceApiProvider
```

Comment:
Do you also want to add this Provider to the TestProvider? I think the tests are failing because of that in the AppConfig tests

Reply:
We don't want to call `renderWithProviders` in this case. Simply calling the standard `render` is sufficient, because AppConfig itself already has the providers.

Reply:
Not just for the AppConfig, I believe we need to have this Provider in the test-utils TestProvider that is used for other tests